File: app/src/main/java/me/srikavin/quiz/model/QuizQuestion.java
package me.srikavin.quiz.model;

import android.media.Image;

import com.google.gson.annotations.Expose;

import java.util.ArrayList;
import java.util.List;

import androidx.annotation.Nullable;

public class QuizQuestion {
    @Nullable
    public Image image;

    @Expose
    public String text;

    @Expose
    public List<QuizAnswer> answers;

    public QuizQuestion() {
        text = "";
        answers = new ArrayList<>();
    }
}

File: app/src/main/java/me/srikavin/quiz/model/QuizQuestion.kt
package me.srikavin.quiz.model

import android.media.Image
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import me.srikavin.quiz.network.common.model.data.QuizQuestionModel
import me.srikavin.quiz.network.common.model.data.ResourceId
import java.util.*

/**
 * A model class representing a quiz question
 */
data class QuizQuestion(
        /**
         * A unique identifier used to represent a specific quiz question
         */
        override val id: ResourceId = ResourceId(UUID.randomUUID().toString()),
        /**
         * The contents of this question (i.e. the question itself)
         */
        @Expose @SerializedName("text") override var contents: String = "",
        /**
         * A list of possible answers to this question
         */
        @Expose override var answers: MutableList<QuizAnswer> = mutableListOf(),
        internal var image: Image? = null
) : QuizQuestionModel
